client/src/test/java/tds/config/TimeLimitConfigurationUnitTests.java

```diff
@@ -0,0 +1,30 @@
+package tds.config;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimeLimitConfigurationUnitTests {
```

Comment:
I've always used the "UnitTests" suffix, but that's solely habit.  Once we land on a convention, I'll rename these files to match.